compute/gpu/how-to/use-nvidia-mig-technology.mdx

```diff
@@ -0,0 +1,347 @@
+---
+meta:
+  title: How to use the NVIDIA MIG technology on GPU Instances
+  description: This section provides information about NVIDIA's MIG technology
+content:
+  h1: How to use the NVIDIA MIG technology on GPU Instances
+  paragraph: This section provides information about NVIDIA's MIG technology
+tags: NVIDIA H100 MIG multi-instance gpu
+dates:
+  validation: 2023-08-31
+  posted: 2023-08-31
+categories:
+  - compute
+---
+
+<Message type="important">
+  * Scaleway offers GPU Instances such as H100 PCIe GPU Instances, RENDER-S GPU Instance, etc.
+  * Nvidia uses the term *"GPU instance"* to designate a MIG partition of a GPU (MIG= Multi-Instance GPU)
+  * To avoid confusion, we will use the term GPU Instance in this document to designate the Scaleway GPU Instance. And we use *"MIG partition"* in the context of the MIG feature.
```

Comment:
```suggestion
  * To avoid confusion, we will use the term GPU Instance in this document to designate the Scaleway GPU Instance, and *"MIG partition"* in the context of the MIG feature.
```